blog/01_payments-pallet-grant.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Payment Pallet: Connecting dots to the real world"
+description: |
+  With the support of the Web3 Foundation Virto brings secure reversible
+  payments to the Polkadot ecosystem
+published_date: 2022-03-01 14:53:00 +0100
+permalink: /blog/payments-pallet-web3-grant
+data:
+  author: Qian & Daniel
+is_draft: true
+---
+
+[pallet]: https://github.com/virto-network/open-runtime-module-library/tree/payments-pallet/payments
+[polkadot]: https://polkadot.network/about
+[kusama]: https://kusama.network
+[lip]: https://virto.network/docs/whitepaper.html
+
+# With the support of Web3 Foundation Grants Program ❤️ 
+
+Virto team is glad to announce the completion of the initial development of its
+[`payments pallet`][pallet], a payments primitive that enables any blockchain
+in the [Polkadot][polkadot] and [Kusama][kusama] ecosystems deliver a simple
+and secure experience when paying for real world goods and services.
+
+>*Web3 Foundation funds research and development teams building the technology
+>stack of the decentralized web. It was established in Zug, Switzerland by
+>Ethereum co-founder and former CTO Gavin Wood. Polkadot is the Foundation's
+>flagship project.*
+
+**Virto Network** aims to build the best decentralized infrastructure for
+marketplaces with local impact in the Polkadot ecosystem. To ensure the success
+of marketplaces, it's Virto's responsibility and pursuit to provide
+marketplaces with secure and convenient payment system. With this goal and
+expectation **Virto Payment Pallet** was born. 
+
+**Virto Payment Pallet** tries to tell people a story of trust, which starts
+with **trustless**. Imagine you want to start your own business, what would be
+the most challenging part? Many answers may already come to mind, among which
+the word **trust** appears often. For example, how could you build the trust
+with your business partners? How could you gain trust from your customers every
+time they purchase from you? How could your trust other services and goods
+whenever you buy? We know it's not easy to start with just naive trust, that's
+why Virto purpose to build the Payment Pallet with the trustless design. People
+could pay each other easily using the pallet. The payment from buyers will
+always be locked first and only be released to sellers once the buyers
+confirmed to have received the product or services. The payment pallet also
+allows the creator of the payment to trigger cancel with a buffer time and the
+recipient to dispute the payment request of sender.   
+
+How does Virto Payments Pallet achieve the desired functionality? As the
```

Comment:
You use different terms for the same concept:

-  Payment Pallet
- Payments Pallet
- payments pallet

It'd be more consistent to use the same term everywhere, whenever the same concept is describe.

Reply:
Thanks, yeah the correct way would be "Payments Pallet", we'll look into that